dev/change_version.py

```diff
@@ -0,0 +1,265 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+This script changes versions in every pom.xml and relevant files.
+
+(How to run)
+python change_version <reef_home> <reef_version_for_pom.xml>  (optional) -s <true or false>   
+
+-s option changes value of 'IsSnapshot' in lang/cs/build.props.
+If you use the option "-s false", bulid.props changes as,
+ <RemoveIncubating>true</RemoveIncubating>
+ <IsSnapshot>false</IsSnapshot>
+ <SnapshotNumber>0</SnapshotNumber>
```

Comment:
I believe so. We need to generate a unique version each time we want to publish a NuGet SNAPSHOT. 99 snapshots between releases seems more realistic than 9 :-)